src/back/controllers/api/models/user.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const User = new Schema({
+    photo: { type: String },
+    firstName: { type: String, required: true },
+    lastName: { type: String, required: true },
+    registerDate: { type: Date, required: true },
+    lastLogin: { type: Date, required: true },
+    rating: { type: Number, required: true },
+    wishList: [{
+        type: Schema.ObjectId,
+        ref: 'Challenge'
+    }],
+    challenges: [{
```

Comment:
This collection contains user related data (status (accepted, complited, etc), date) together with ref to a document in existing challanges collection